docs/mast/mast.rst

```diff
@@ -636,13 +551,7 @@ the user that they might be getting a subset of the true result set.
     50180585 0.003984902849540913  322.4931746094701 ...          nan          0
      8150896 0.006357935819940561 322.49334740450234 ...          nan          0
    100906349  0.00808206428937523  322.4932839715549 ...          nan          0
-   105434103 0.011947078376104195 322.49324000530777 ...          nan          0
-   103116183  0.01274757103013683  322.4934207202404 ...          nan          0
-    45593349 0.013026569623011767  322.4933878707698 ...          nan          0
-   103700905  0.01306760650244682  322.4932769229944 ...          nan          0
-   102470085 0.014611879195009472 322.49311034430366 ...          nan          0
-    93722307  0.01476438046135455 322.49348351134466 ...          nan          0
-    24781941 0.015234351867433582 322.49300148743345 ...          nan          0
+   ...
```

Comment:
e.g. this example explicitly prints out 10 lines, so I would keep all those 10 lines. 

The other changes where there is a longer ellipse, sure, culling those back sounds fine.

Reply:
Would it be okay to change the example to explicitly print 3 lines instead? A lot of the examples I shortened also use 10.

Reply:
@jaymedina should comment on that. 
I feel the 10 is more reasonable for a use case, but I don't really mind either case, just don't use ellipses for these short examples.